client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -57,7 +59,10 @@ class ElementEditor extends PureComponent {
   handleDragEnd(sourceId, afterId) {
     const { actions: { handleSortBlock }, areaId } = this.props;
 
-    handleSortBlock(sourceId, afterId, areaId);
+    handleSortBlock(sourceId, afterId, areaId).then(() => {
+      const preview = window.jQuery('.cms-preview');
```

Comment:
Yes that's reasonable and I agree. While I was doing this I was thinking it would be nice to consolidate a lot of this "entwine hack" code into some `lib` js file. Preferably available in admin or something. There's a bunch of places we use this code now.